### ISSUE TITLE
having error while format mimi.types

### ISSUE THREAD

**Issue by Taymindis** [nginxinc/crossplane#58]
```bash
crossplane format /etc/nginx/mime.types
Traceback (most recent call last):
  File "/usr/local/bin/crossplane", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/crossplane/__main__.py", line 228, in main
    func(**kwargs)
  File "/usr/local/lib/python3.7/site-packages/crossplane/__main__.py", line 134, in format
    raise NgxParserBaseException(e['error'], e['file'], e['line'])
crossplane.errors.NgxParserBaseException: "types" directive is not allowed here in /etc/nginx/mime.types:5 in /etc/nginx/mime.types:5
```

**Comment by aluttik** [comment]
crossplane format currently only works on the main config file for some reason. I'll change it and keep you updated.

**Comment by aluttik** [comment]
@Taymindis: this has been fixed and crossplane 0.4.15 has been uploaded to PyPI :+1:
